pkgs/applications/editors/vim/plugins/build-vim-plugin.nix

```diff
@@ -9,7 +9,8 @@
 
 rec {
   buildVimPlugin = attrs@{
-    name ? "${attrs.pname}-${attrs.version}",
+    pname,
+    version ? "«version-missing»",
```

Comment:
dont we want to make it mandatory as well ?

Reply:
does "«" even work in path ?

Reply:
idk, this is what you proposed in the function in the issue. I don't think we want this mandatory because the version should not be tracked by hand. e.g. I get vimPlugins from flake inputs, I don't wanna change versions every time I update those. 
This is specifically for cases where you don't have an auto-generated plugin. We can have a  different string `"«version-missing»"` though.

Reply:
> I get vimPlugins from flake inputs, I don't wanna change versions every time I update those.

Ha interesting, I was wondering how you were building your plugins. 
version ? "«version-missing»" triggers a `store path '8hb43ah1gc5vkqyxzln2j1739lyf8rbm-vimplugin-vim-fugitive-«version-missing»' contains illegal character ''`.

> I don't wanna change versions every time I update those.

you dont have to, do you ? `version = "master"` or you can set it from the flake input src. Are your dotfiles public ?

Reply:
Just look at how it's done in the manual. That's exactly how many people do it. Which makes sense because it's in the manual. But yeah I guess that I could also set it to master. Just don't know why it'd be necessary then

Reply:
```suggestion
    version ? "unknown",
```